stackslib/src/net/download/epoch2x.rs

```diff
@@ -33,7 +33,6 @@ use stacks_common::util::{get_epoch_time_ms, get_epoch_time_secs, log};
 use crate::burnchains::{Burnchain, BurnchainView};
```

Comment:
No need to review this file. It's just a renamed `stackslib/src/net/download.rs`.